### ISSUE TITLE
Charting quirk

### ISSUE THREAD

**Issue by dazfuller** [microsoft/azuredatastudio#2954]
Issue Type: <b>Bug</b>

I was just looking at charting to series in Azure Data Studio, when selecting "line" as an option the visualization looks nothing like what I'd expect with all of the data stacked.

![image](https://user-images.githubusercontent.com/204848/47163995-00f6ef80-d2ef-11e8-9345-71ba2407333e.png)

However if I change the chart type to "Bar", change "Data Direction" to "Vertical" and then switch back to Line it looks perfect.

![image](https://user-images.githubusercontent.com/204848/47164012-0ce2b180-d2ef-11e8-8938-da74a69d36ac.png)

To re-create produce a result set that looks something like

Id | Val1 | Val2
-- | ---- | ----
1 | 1.1 | 1.2
2 | 2.1 | 0.94

etc...

Azure Data Studio version: azuredatastudio 1.0.0 (cab8f3e37c9ac4367e0dacef5185da08bd8021c2, 2018-09-20T22:04:35.897Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|7.93GB (1.28GB free)|
|Process Argv|C:\Program Files\Azure Data Studio\azuredatastudio.exe|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by rgranadino** [comment]
I ran into this issue as well. It also seems that once the chart is out of view (e.g. switching to another tab) it "resets". Toggling between chart types (e.g. pie, then back to line) will fix it without having to change the "Data Direction". 

This is the query I used to generate a set:

`select date(d) as day, ceil(random() * 1000) count
from generate_series(
    current_date - interval '30 day', 
    current_date, 
  '1 day'
) d `

![AzureDataStudioLineGraphBug](https://user-images.githubusercontent.com/327608/55028984-3ef6ee80-4fc6-11e9-98f2-c32a7a6e9739.gif)

Version: 1.5.1
